setupRadio.js

```diff
@@ -0,0 +1,30 @@
+function setupRadio(x, y, radioOptions, verticalFlag, defaultOption) {
+  var BUTTON_WIDTH = 30;
+  var labelFontSize = floor(height / 75);
+  var thisRadio = createRadio();
+  var longestOptionLength = 0;
+  var radioWidth;
+
+  for (var i = 0; i < radioOptions.length; i++) {
+    var option = radioOptions[i];
+    var optionLength = textWidth(option);
+
+    thisRadio.option(option, i);

```

Comment:
So, starting this at 0 does seem to create some issues. It seems to be a bug with `p5.js`, so if you could create a simplified example and report an issue on their GitHub I'm sure that'd get your name out there ;)

https://github.com/processing/p5.js/issues


For our use, we can just start at 1, most easily achieved by just making this line `thisRadio.option(option, i + 1)`